CHECKLIST.md

```diff
@@ -0,0 +1,107 @@
+# Platform Release Checklist
+
+## Dev
+
+- [ ] `./pear.dev sidecar shutdown`
```

Comment:
after this, add `git clean -x -f -d` as the next step and always do npm run bootstrap